interface/patient_file/summary/immunizations.php

```diff
@@ -108,6 +111,10 @@
         $result = sqlQuery($sql, array($_GET['id']));
 
         $administered_date = new DateTime($result['administered_date']);
+        $uuid = null;
+        if (isset($result['uuid']) && UuidRegistry::isValidUUID($result['uuid'])) {
+            $uuid = UuidRegistry::uuidToString($result['uuid']);
+        }
```

Comment:
To clarify here, are we checking if the uuid is empty or valid? (I'd rec empty since would hate to overwrite a current uuid if the underlying code decided that the current uuid's were not valid in future).

Reply:
It would be isEmptyBinary as its a Binary value :)